### PR TITLE
fix:test

### PR DESCRIPTION
**Pull request by hori-tenma**
test

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -173,6 +173,8 @@ <h4>Git実践入門 (@syobochimさん)</h4>
     <script src="assets/js/util.js"></script>
     <script src="assets/js/main.js"></script>
 
+
+    <!-- test -->
 </body>
```
Comment (hori-tenma) [inline index.html:179]: review test  comment

**Review by hori-tenma — COMMENTED** [review]
please review

File: index.html (hunk 1 above)
Comment (kzh-amk-928) [reply index.html:179]: OK

**Review by kzh-amk-928 — APPROVED** [review]
OK

**Pull request merged**